### PR TITLE
fix some grammar

### PR DESCRIPTION
**Pull request by KristofferC**

### CODE REVIEW

File: docs/src/sysimages.md [hunk 1]
```diff
@@ -2,15 +2,15 @@
...
+A sysimage is a file that, in a loose sense, contains a Julia session
+serialized to a file. A "Julia session" includes things like loaded packages,
+global variables, inferred and compiled code, etc. By starting Julia with a
 sysimage, the stored Julia session is deserialized and loaded. The idea behind
 the sysimage is that this deserialization is faster than having to reload
 packages and recompile code from scratch.
 
 Julia ships with a sysimage that is used by default when Julia is started. That
-sysimage contains the Julia compiler itself, the standard libraries and also
+sysimage contains the Julia compiler itself, the standard libraries, and also
```
Comment (Sacha0) [inline docs/src/sysimages.md:13]: RIP Oxford comma

File: docs/src/sysimages.md [hunk 2]
```diff
@@ -120,8 +120,8 @@ has been replaced, the next `create_sysimage` call will create a new sysimage
 based on the replaced sysimage. It is possible to create a sysimage
 non-incrementally by passing the `incremental=false` keyword. This will create
 a new system image from scratch. However, it will lose the special
-precompilation that the Julia bundled sysimage provides which is what make the
-REPL and package manager not require compilation after a Julia restart.. It is
+precompilation that the Julia bundled sysimage provides which is what makes the
+REPL and package manager to not require compilation after a Julia restart. It is
```
Comment (Sacha0) [inline docs/src/sysimages.md:124]: This snippet reads more naturally to me without the additional `to`. Tangentially, perhaps add a comma between `provides` and `which`, i.e.
```suggestion
precompilation that the Julia bundled sysimage provides, which is what makes the
REPL and package manager not require compilation after a Julia restart. It is
```

File: docs/src/sysimages.md [hunk 3]
```diff
@@ -159,7 +159,7 @@ Example.hello("friend")
 
 We now create a new system image called `ExampleSysimagePrecompile.so`, where
 the `precompile_execution_file` keyword argument has been given, pointing to
-the file just shown above:
+the file that was just shown above:
```
Comment (Sacha0) [inline docs/src/sysimages.md:162]:
```suggestion
the file shown just above:
```
perhaps, but 🤷 :)

**Review by Sacha0 — APPROVED** [review]
Great touchups! :)

**Comment by Sacha0** [comment]
(Sorry @kmsquire! I did not see your restoration before deleting the branch; restored again :).)

**Comment by kmsquire** [comment]
No worries—I was just checking some of the files on the branch, and forgot to go back and delete it. Cheers!

**Pull request merged**